cms/bundles/tests/test_wagtail_hooks.py

```diff
@@ -152,17 +151,24 @@ def test_add_to_bundle_buttons(self):
 
     def test_add_to_bundle_buttons__doesnt_show_for_pages_in_bundle(self):
         """Checks that the button doesn't appear for pages already in a bundle."""
-        release_calendar_page = ReleaseCalendarPageFactory()
+        article_series_page = ArticleSeriesPageFactory()
         contexts = [
-            (reverse("wagtailadmin_pages:edit", args=[release_calendar_page.id]), "header"),
-            (reverse("wagtailadmin_explore", args=[release_calendar_page.get_parent().id]), "listing"),
+            (
+                reverse("wagtailadmin_pages:edit", args=[article_series_page.id]),
+                reverse("bundles:add_to_bundle", args=[article_series_page.id]),
+                "header",
+            ),
+            (
+                reverse("wagtailadmin_explore", args=[article_series_page.get_parent().id]),
+                reverse("bundles:add_to_bundle", args=[article_series_page.id]),
+                "listing",
+            ),
         ]
         BundlePageFactory(parent=self.draft_bundle, page=self.statistical_article_page)
 
         self.client.force_login(self.publishing_officer)
 
-        for url, context in contexts:
+        for url, add_to_bundle_url, context in contexts:
             with self.subTest(msg=f"Non-bundleable page - {context}"):
                 response = self.client.get(url)
-                self.assertNotContains(response, "Add to Bundle")
-                self.assertNotContains(response, self.add_to_bundle_url)
+                self.assertNotContains(response, add_to_bundle_url)
```

Comment:
Note: I've removed the check for "Add to Bundle" as this could still appear as an option for the parent in the dropdown in the header. Checking for the `add_to_bundle_url` seems sufficient to validate this piece of functionality.